src/Cech_complex/include/gudhi/MEB_filtration.h

```diff
@@ -46,7 +46,15 @@ void assign_MEB_filtration(Kernel&&k, SimplicialComplexForMEB& complex, PointRan
   std::vector<Point_d> pts;
   CGAL::NT_converter<FT, Filtration_value> cvt;
 
+  // This block is only needed to get ambient_dim
+  if(std::begin(points) == std::end(points)) {
+    // assert(complex.is_empty());
+    return;
+  }
+  int ambient_dim = k.point_dimension_d_object()(*std::begin(points));
+
   auto fun = [&](Simplex_handle sh, int dim){
+    using std::max;
```

Comment:
I would keep std::max everywhere else max is called to be more explicit, but maybe a matter of taste. You can forget this comment if you do not agree

Reply:
Note that in this function, I want `max` to be found by ADL if possible, I do not want to use std::max if the type is `CGAL::Lazy_exact_nt` for instance (it should use `CGAL::max` in that case). I still need to make `std::max` available in case the type is built-in, like `double`, hence the `using`.
I could copy the using directive before each call to `max` instead of once for the whole lambda, but there are 4 calls spread in the various branches, so it would be noisy.

Reply:
Ok, I got it, thanks for the precision